Program.cs

```diff
@@ -75,68 +71,90 @@ public static void UpdatePerson()
                         case 1:
```

Comment:
You should wrap these 1 and 2. You can make them `const int` field. That would be more readable.